src/screens/FinancialSummary/ConfirmPayment.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import { Button, Typography } from '@material-ui/core';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+import FinancialInfo from './components/FinancialInfo';
+import PhotoLibraryIcon from '@material-ui/icons/PhotoLibrary';
+import { FinancialSummaryRecord } from '../../lib/airtable/interface';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    content: {
+      color: theme.palette.text.primary,
+    },
+    confirmButton: {
+      display: 'flex',
+      margin: '0 auto',
+      borderColor: theme.palette.primary.main,
+      borderRadius: '18px',
+      backgroundColor: theme.palette.primary.main,
+      color: theme.palette.common.white,
+      letterSpacing: '0.14em',
+      height: '41px',
+      width: '70%',
+      marginTop: 15,
+    },
+    cameraButton: {
+      backgroundColor: '#F7F9FC',
+      width: '95%',
+      height: 130,
+      margin: 10,
+      border: '3.5px dashed ' + theme.palette.divider,
+      radius: '6px',
+    },
+    cameraText: {
+      color: theme.palette.primary.main,
+    }
+});
+
+interface ConfirmPaymentProps extends RouteComponentProps {
+  classes: { content: string; confirmButton: string; cameraButton: string; cameraText: string; };
+  financialSummary: FinancialSummaryRecord;
+  location: any;
+}
+
+function ConfirmPayment(props: ConfirmPaymentProps) {
```

Comment:
Name isn't descriptive enough. It doesn't tell enough about what this component is (ideally, i'd be able to match a specific component like this one to the screen it's on just by looking at it's name). For components especially, longer names are better (unless it's egregiously long). I wouldn't mind something along the lines of `FinancialSummaryPayment` (which isn't really perfect either)